source/tutorial/acceptance-test.md

```diff
@@ -78,35 +78,37 @@ Our launched Chrome web browser now shows 9 successful tests. If you toggle the
 
 As mentioned before, our initial test just made sure everything was running properly. Now let's replace that test with the list of tasks we want our app to handle (described up above).
 
-```/tests/acceptance/list-rentals-test.js{+6,+7,+8,+9,+10,+11,+12,+13,+14,+15,+16,+17,+18,+19,+20,+21,+22,-23,-24,-25,-26,-27,-28,-29}
-import { test } from 'qunit';
-import moduleForAcceptance from 'super-rentals/tests/helpers/module-for-acceptance';
+```/tests/acceptance/list-rentals-test.js{+8,+9,+10,+11,+12,+13,+14,+15,+16,+17,+18,+19,+20,+21,+22,+23,+24,+25,-26,-27,-28,-29}
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
 
-moduleForAcceptance('Acceptance | list-rentals');
+module('Acceptance | list-rentals', function(hooks) {
+  setupApplicationTest(hooks);
 
-test('should show rentals as the home page', function (assert) {
-});
+  test('should show rentals as the home page', function (assert) {
```

Comment:
The blueprints will generate these as `async` functions. Should you start with them as async here?